serverless.yml

```diff
@@ -250,21 +250,12 @@ resources:
             AttributeType: S
           - AttributeName: username
             AttributeType: S
-          - AttributeName: user_id
-            AttributeType: S
         BillingMode: PAY_PER_REQUEST
         KeySchema:
           - AttributeName: partition_key
```

Comment:
is this still correct? do we still need this attribute partition_key USER? I was not sure... @larsuzh

Reply:
Yes, this should be working. (If I understood it correctly) We write it to the DB in this way in order to distinguish between users and follow requests, which are also in this table